.github/pull_request_template.md

```diff
@@ -1,22 +1,26 @@
-<!-- Thank you for submitting a pull request! Find more information in our development guide at https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md and contact us on #dev in Slack. -->
-<!-- Provide an overview of the PR for maintainers and reviewers. -->
+<!-- Thank you for submitting a pull request! Find more information 
+at https://trino.io/development/process.html, 
```

Comment:
The other thing worth considering is that we could potentially change this to markdown so the links work .. but then it ends up being text that shows up in the PR after .. 

Otherwise the flow you are suggesting for issues could help with links to info @wendigo